config.mak

```diff
@@ -145,6 +145,8 @@ export OE_OESIGN_TOOL_PATH := $(OE_SDK_ROOT)/bin
 
 OPENENCLAVE = ${OE_SDK_ROOT}/lib/openenclave/enclave/liboeenclave.a
 
+SGXLKL_CFLAGS_EXTRA+=-DOE_WITH_EXPERIMENTAL_EEID
```

Comment:
Isn't this flag specified at OE compile time? By the time you're building SGX-LKL, I think it's too late.

Reply:
Oh, I guess it's also needed here for headers

Reply:
That's right, I need it for the headers. But yes, I also need it for the OE build, but I haven't checked whether I can enable it just for this branch for now.

Reply:
If you modify the pipelines file it shouldn't affect any other CI runs until this PR is merged.